src/style.css

```diff
@@ -0,0 +1,24 @@
+.first {
+  background: red;
+  height: 100px;
+  width: 100%;
+  margin: 0;
```

Comment:
width: 100%;
  margin: 0; - defaults for div, do not need to specify these styles